### PR TITLE
Integrate Grid Snapping with Copy/Paste Feature

### LINKED ISSUE #324: Integrate Grid Snapping with Copy Paste Feature
Related to https://github.com/bpmn-io/bpmn-js/issues/973.

### PR DESCRIPTION
**Pull request by philippfromme**
Closes #324.

### CODE REVIEW

File: lib/features/grid-snapping/behavior/PasteBehavior.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        tree = context.tree;
+
+    if (!tree[0]) {
+      return;
+    }
+
+    var mid = getMid(getBBox(tree[0]));
+
+    // snap paste position but maintain each element's grid offset
+    position.x = gridSnapping.snapValue(position.x) + (mid.x % 10);
```
Comment (barmac) [inline lib/features/grid-snapping/behavior/PasteBehavior.js]: Can we use constant for offset?

Comment (barmac) [reply lib/features/grid-snapping/behavior/PasteBehavior.js]:
```suggestion
    position.x = gridSnapping.snapValue(position.x) + (mid.x % GRID_OFFSET);
```

Comment (philippfromme) [reply lib/features/grid-snapping/behavior/PasteBehavior.js]: Yeah 👍

**Comment by philippfromme** [comment]
@barmac ready for round two.

**Pull request merged**